pkg/kafka/aclutil/enums.go

```diff
@@ -155,18 +144,9 @@ func GetMappedResourceTypeFilterValue(resourceType string) kafkainstanceclient.A
 	return resourceTypeFilterMap[resourceType]
 }
 
-// getValidOperationsResponseMap returns a map of the ACL
-// resource operations to the values used in this package
-// eg: "ALTER": "alter"
-func getValidOperationsResponseMap() map[string]string {
-	reverseOperationMap := GetReversedOperationMap()
-	// The API is returning internal Kafka operations, e.g describe_configs
-	// It will change to align with the API enums, eg: DESCRIBE_CONFIGS
-	// This will provide backwards compatibility for both and can be removed in a later release
-	for op, v := range reverseOperationMap {
-		validOperationsResponseMap[string(op)] = string(v)
-	}
-	return validOperationsResponseMap
+// GetResourceTypeFilterKeyMap gets the mappings for ACL operations
+func GetResourceTypeFilterKeyMap() map[string]string {
```

Comment:
```suggestion
func GetResourceTypeOperationKeyMap() map[string]string {
```
This seems to be unused now though.

Reply:
Thanks!